sfcasts/prod-environment.md

```diff
@@ -5,18 +5,17 @@ Open the `.env` file in the root of our project and change this `dev` environmen
 [[[ code('76ca37e8e8') ]]]
 
 ***TIP
```

Comment:
Can we use something more forceful than TIP here? Maybe CAUTION or WARNING?

Reply:
So far we add all notes like this with TIP syntax... We also have SEEALSO that's something useful when you want to point to a page with more details... and that's it, we only have these 2.

So, in general, I think this TIP, along with a video note, should be OK in this case